instrumentation/methods/javaagent/src/declarativeConfigTest/resources/declarative-config.yaml

```diff
@@ -0,0 +1,33 @@
+file_format: "0.4"
+tracer_provider:
+  processors:
+    - simple:
+        exporter:
+          memory:
+    - simple:
+        exporter:
+          console:
+
+logger_provider:
+  processors:
+    - simple:
+        exporter:
+          memory:
+
+meter_provider:
+  readers:
+    - periodic:
+        # Set really long interval. We'll call forceFlush when we need the metrics
+        # instead of collecting them periodically.
+        interval: 1000000
+        exporter:
+          memory:
```

Comment:
i hope that eventually the testing extension will somehow configure these

Reply:
created https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/14148